src/page-modules/departures/stop-place/index.tsx

```diff
@@ -115,7 +137,8 @@ export function EstimatedCallList({ quay }: EstimatedCallListProps) {
             <button
               className={and(style.listItem, style.listItem__last)}
               aria-label={t(Departures.stopPlace.quaySection.a11yToQuayHint)}
-              onClick={() => alert('Not implemented yet')} // @TODO: implement function
+              onClick={getMoreDepartures}
+              disabled={isFetchingDepartures}
```

Comment:
Should it possibly be a loading state as well, so it shows that more departures are being fetched?

Reply:
Let me rephrase, since the answer to the previous question is of course that's what `isFetchingDepartures` is.

Rephrased: should we show a spinner or something that indicates that departures are being fetched? Just disabling the button might not be indicative enough. Also a11y labels.

Reply:
> should we show a spinner or something that indicates that departures are being fetched? 

Yes, I think that's better. I've updated to show a spinner while fetching.

> Also a11y labels.

Do you mean a11y label when the button is disabled? As in changing the current `aria-label` when `isFetchingDepartures` is true?

Reply:
Ah, you were probably thinking of `aria-disabled`. That's also fixed now.

Reply:
`aria-disabled` should be set to either `true` or `false` from what I can see. With a11y labels I meant that it should be clear to the user when the loading spinner is showing (and the departures are being fetched), and by only disabling the button it might not be obvious.

Reply:
With the current change the `aria-disabled` is set to `true` when the state is `loading` (done by the Button component).

Reply:
Isn't it set to `aria-disabled={'DISABLED'}`? That's what I see. 🤔

Reply:
Sorry about the confusion! I pushed some changes that I was unaware of.